.build_install.sh

```diff
@@ -1,18 +1,19 @@
 #!bin/bash
 
 function build_install() {
-  pushd $1
-  pip install -r requirements.txt
-  mkdir build
-  pushd build
-  cmake .. -DENABLE_PYTHON=ON \
+  local src=$1
+
+  mkdir -p $src/build
+  pip install -r $src/requirements.txt
+
+  cmake -S $src -B $src/build -GNinja -DENABLE_PYTHON=ON \
 -DBUILD_APPLICATIONS=ON \
 -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
 -DCMAKE_PREFIX_PATH=$INSTALL_DIR \
 -DCMAKE_INSTALL_NAME_DIR=$INSTALL_DIR/lib 
-  make install
-  popd
-  popd
+  ninja -C $src/build install
+
+  rm -rf $src
```

Comment:
This script has a lot of dependents in other repos, think it is best to leave out the ninja stuff :disappointed:

Reply:
:cry:

Reply:
should the ninja stuff go as @sondreso points out?